client/src/styles/slider.css

```diff
@@ -67,4 +67,7 @@
   .slider-menu {
     margin-top: -4.5em;
   }
+  .sa-text__promo {
+    margin-top: 5em;
```

Comment:
Can you check to make sure this still appears well on the web version(middle) and middle on mobile?